tfhe/src/core_crypto/commons/ciphertext_modulus.rs

```diff
@@ -298,10 +298,8 @@ mod tests {
         assert!(std::mem::align_of::<CiphertextModulus<u128>>() == std::mem::align_of::<u128>());
 
         {
-            let mod_32_res = CiphertextModulus::<u32>::try_new_power_of_2(32);
-            assert!(mod_32_res.is_ok());
```

Comment:
why was this modified ? it has nothing to do with the current PR right ?

Reply:
I did this change while removing `assert!(*.is_ok());`s from another version of the PR
I think it's better this way
I'll put that it a separate commit

Reply:
I would prefer if we avoid such changes which are unrelated to the ongoing work, it makes the PRs more noisy and delays the reviews :)

Reply:
Done